dev-docs/bidders/adport.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: bidder
+title: AdPort
+description: Prebid AdPort Bidder Adapter
+aliasCode: adverxo
+pbjs: true
```

Comment:
I would recommend setting this and  `pbs_app_supported`  to `false` ( in all files ). Then we can already merge this.

You then would open a second PR that sets those values to true and link the prebid-server pull request. This way we aren't blocked by the PBS review

Reply:
Hi @muuki88, thanks for your recommendation. We've already updated the doc files with the changes to move forward with pbjs.